test/integration/tasks.test.js

```diff
@@ -112,6 +113,67 @@ describe("Tasks", function () {
           return done();
         });
     });
+    it("Should return 400 if assignee username does not exist", function (done) {
+      chai
+        .request(app)
+        .post("/tasks")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .send({
+          title: "Test task - Create",
+          type: "feature",
+          endsOn: 123,
+          startedOn: 456,
+          status: "completed",
+          percentCompleted: 10,
+          completionAward: { [DINERO]: 3, [NEELAM]: 300 },
+          lossRate: { [DINERO]: 1 },
+          assignee: "dummyUser",
+          participants: [],
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(400);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User not found");
+          return done();
+        });
+    });
+    it("Should not return invalid username in participants", function (done) {
```

Comment:
why aren't we checking for valid usernames in participants array?

Reply:
As per my discussion with @ankushdharkar, we decided to filter out invalid usernames if passed in the participants array and throw a 400 Bad Request response if invalid username is passed as an assignee

Reply:
okay.